cms/templates/partials/card_details_top.html

```diff
@@ -1,9 +1,9 @@
-{% load static humanize wagtailcore_tags wagtailimages_tags image_version_url %}
+{% load static humanize wagtailcore_tags wagtailimages_tags wagtail_image_lazy_load %}
 
 <div class="cover-image">
   <div class="cover-image-frame">
     {% if courseware_page.thumbnail_image %}
-      <img src="{% image_version_url courseware_page.thumbnail_image "fill-275x155" %}" alt="{{ courseware_page.thumbnail_image.title }}" />
+      {% wagtail_image_lazy_load courseware_page.thumbnail_image fill-275x155 alt=courseware_page.thumbnail_image.title class="wagtail-lazy-load" %}
```

Comment:
Ideally we wouldn't need to specify the same class for every one of these images. It would be nice to have a default class and the option to extend. A param like `addedClass` might make sense. Also, the class can just be called `lazy` instead of `wagtail-lazy-load`

Reply:
I do agree with your suggestions but I am trying to keep it as simple as we can. The issue is with the template tag we are using, when we create a simple_tag `@register.simple_tag()` it works as a function to which you can pass values and do some **kwargs stuff but in our case, we have to use `@register.tag(name="xyz")` as the function have to call another [wagtail's tag named **image**](https://github.com/mitodl/mitxpro/blob/hamza/2065-image_lazy_loading/cms/templatetags/wagtail_lazy_image.py#L61).
Both the tags have different parsing techniques. So adding a new attribute makes the whole process complicated and messy.
As far as changing the name of `wagtail-lazy-load` to `lazy` is concerned, I have updated that.